ui/components/src/StatChart/StatChart.tsx

```diff
@@ -117,17 +125,15 @@ export function StatChart(props: StatChartProps) {
         {formattedValue}
       </Typography>
       {sparkline !== undefined && (
-        <Box sx={{ flex: 1 }}>
```

Comment:
@sjcobb curious if there was a reason you added `flex: 1` here... i removed it because i didn't notice any difference but maybe i'm missing something

Reply:
I think I did this ages ago to fix some css issues. With the `EChart` component having height set to `100%`, I think you're getting the same effect, so it should be ok to adjust it.